dhall/src/Dhall/Version.hs

```diff
@@ -0,0 +1,12 @@
+module Dhall.Version ( dhallVersion
```

Comment:
I think it would be good to document, that this module is about the version of the `dhall` library, not about the version of the Dhall standard.